frontend_server/lib/server.dart

```diff
@@ -45,7 +47,14 @@ ArgParser _argParser = new ArgParser(allowTrailingOptions: true)
           ' Intended use is to satisfy different loading strategies implemented'
           ' by gen_snapshot(which needs platform embedded) vs'
           ' Flutter engine(which does not)',
-      defaultsTo: true);
+      defaultsTo: true)
+  ..addFlag('track-widget-creation-locations',
```

Comment:
DBC: this flag name is long; perhaps `--track-widget-creation`? `--track-widgets`?

Reply:
--track-widget-creation is much better. Updated here and elsewhere.